### ISSUE TITLE
release/latest-*.txt are invalid builds

### ISSUE THREAD

**Issue by zmerlynn** [kubernetes/release#51]
Something (probably Anago?) is trashing the "latest.txt" files in the `release` gsutil bucket and making them useless for Jenkins. It's probably been happening for a while, but only the upgrade tests actually care, and they've been woefully ignored.

This is not supposed to produce a build that doesn't exist in GCS:
```
$ gsutil cat gs://kubernetes-release/release/latest-1.3.txt
v1.3.5-beta.0
```

I'm going to shift the Jenkins builds over to the CI versions instead, but it should be possible to use the release bucket.

**Comment by zmerlynn** [comment]
cc @fabioy @maisem

**Comment by zmerlynn** [comment]
Actually, even `gs://kubernetes-release-dev/ci/latest-1.2.txt` wrong right now, which is what Jenkins was attempting to pull.

**Comment by zmerlynn** [comment]
Oh, sorry! `v1.2.7-beta.0` exists in `gs://kubernetes-release-dev/ci`. It's getting pushed to the wrong place.

**Comment by zmerlynn** [comment]
False alarm. These releases totally exist, in both buckets.

**Comment by zmerlynn** [comment]
I see what's wrong now.

```
zml@zml:~/git$ gsutil ls gs://kubernetes-release-dev/ci/v1.2.7-beta.0/ | cut -f5- -d/ | sort > /tmp/ci
zml@zml:~/git$ gsutil ls gs://kubernetes-release/release/v1.2.7-beta.0/ | cut -f5- -d/ | sort > /tmp/release
$ diff /tmp/ci /tmp/release
1,2c1,2
< v1.2.7-beta.0/bin/
< v1.2.7-beta.0/extra/
---
> v1.2.7-beta.0/darwin/
> v1.2.7-beta.0/gce/
```

We launch the `release` version in GKE (because `alpha` and `beta` releases come from the release bucket), and it's missing the `extra` directory.

**Comment by zmerlynn** [comment]
I've manually corrected the `v1.3.5-beta.0` and `v1.2.7-beta.0` release directories. I'm leaving this bug open for analysis/tool correction.
